parachain/runtime/litentry/src/precompiles.rs

```diff
@@ -118,6 +119,12 @@ pub type PrecompilesSetAt<R> = (
 	PrecompileAt<AddressU64<1026>, ECRecoverPublicKey, (CallableByContract, CallableByPrecompile)>,
 	PrecompileAt<AddressU64<1027>, Ed25519Verify, (CallableByContract, CallableByPrecompile)>,
 	// Litentry precompiles (starts from 0x5000):
+	// ParachainStaking: pallet_parachain_staking = 15 + 20480
```

Comment:
Hmm, why not 11 + 20480 to make it consistent? 🤔

Reply:
corrected